tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreRuntimeVersion60Tests.cs

```diff
@@ -71,6 +72,122 @@ public async Task CanBuildAndRun_NetCore60MvcApp()
+                .AddCommand($"export {agentExtensionVersionEnv}=~3")
+                .AddCommand($"export {connectionStringEnv}=alkajsldkajd")
+                .AddCommand(
+                $"oryx create-script -appPath {appOutputDir} -bindPort {ContainerPort}")
+                .AddCommand("cat run.sh")
+                .AddStringExistsInFileCheck(OryxAppInsightsAttachString1, $"run.sh")
+                .AddStringExistsInFileCheck(OryxAppInsightsAttachString2, $"run.sh")
+                .AddCommand(DefaultStartupFilePath)
+                .ToString();
+
+            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
+                NetCoreApp60MvcApp,
+                _output,
+                new DockerVolume[] { volume, appOutputDirVolume },
+                _imageHelper.GetGitHubActionsBuildImage(),
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    buildImageScript
+                },
+                _imageHelper.GetRuntimeImage("dotnetcore", "6.0"),
+                ContainerPort,
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    runtimeImageScript
+                },
+                async (hostPort) =>
+                {
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
+                    Assert.Contains("Welcome to ASP.NET Core MVC!", data);
+                });
+        }
+
+        [Fact]
+        public async Task CanBuildAndRun_DoesNot_Add_Oryx_AppInsights_Codeless_Configuration()
+        {
+            // Arrange
+            var dotnetcoreVersion = DotNetCoreRunTimeVersions.NetCoreApp60;
+            var hostDir = Path.Combine(_hostSamplesDir, "DotNetCore", NetCoreApp60MvcApp);
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var appDir = volume.ContainerDir;
+            var appOutputDirVolume = CreateAppOutputDirVolume();
+            var appOutputDir = appOutputDirVolume.ContainerDir;
+            var agentExtensionVersionEnv = ExtVarNames.UserAppInsightsAgentExtensionVersion;
+            var connectionStringEnv = ExtVarNames.UserAppInsightsConnectionStringEnv;
+            var OryxAppInsightsAttachString1 = "export ASPNETCORE_HOSTINGSTARTUPASSEMBLIES=";
+            var OryxAppInsightsAttachString2 = "export DOTNET_STARTUP_HOOKS=";
+
+            var buildImageScript = new ShellScriptBuilder()
+               .AddDefaultTestEnvironmentVariables()
+               .AddCommand(
+                $"oryx build {appDir} -i /tmp/int --platform {DotNetCoreConstants.PlatformName} " +
+                $"--platform-version {dotnetcoreVersion} -o {appOutputDir}")
+               .ToString();
+            var runtimeImageScript = new ShellScriptBuilder()
+                .AddCommand($"export {agentExtensionVersionEnv}=~2")
+                .AddCommand($"export {connectionStringEnv}=alkajsldkajd")
```

Comment:
We don't need add these two lines right? We want to test if `scriptGenerator.go ` set these up for the runtime. Could you try without these two lines? Same for other tests

Reply:
i wanted to keep this as a negative test .. i wanted to check if agentextensionversionenv has value anything else as ~3

Reply:
Then how can we test `scriptGenerator.go` file to see if it set up correctly, could you remove these lines in one of the positive tests

Reply:
i dont understand ... let me explain what i understood ..
1. we already have tests where we dont set the environment variables in .net 6 runtime, so those tests will cover the scenario that if we dont set these env variables we dont trigger the codeless attach logic.
2. I added this test specifically to set the environment variables with wrong value so that we can check setting with wrong values doesn't trigger the codeless attach logic
3.  We have a positive test where we set the env variables with correct value to trigger the codeless attach logic

Reply:
I got the first two scenerios, thanks! It's only the third one I'm thinking of, please look at the other comment I just added